app/src/main/java/umc/cozymate/ui/splash/SplashActivity.kt

```
@@ -124,22 +119,17 @@ class SplashActivity : AppCompatActivity() {
+                try {
+                    splashViewModel.setTokenInfo(result.body()!!.result.tokenResponseDTO)
+                    splashViewModel.saveToken()
+                    splashViewModel.memberCheck()
+                    splashViewModel.memberInfoResponse.observe(this) { memberInfo ->
+                        if (memberInfo != null) {
+                            splashViewModel.saveUserInfo(memberInfo)
+                            goCozyHome()
+                        } else goUnivCert()
                     }
-                } else {
+                } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)